src/content/experimental/ui-shell/usage.md

```diff
@@ -87,14 +124,26 @@ The Side-nav contains secondary navigation and fits below the header. It can be
 
 #### Flexible Side-nav
 
-The Flexible Side-nav has both an expanded and collapsed state. The expanded state contains both icons and corresponding L2 text, while the collapsed state shows only the icon. The side-nav expands on hover. It can also be expanded (to a fixed state) or collapsed by clicking on the chevron icon at the bottom of the panel.
+Use a Flexible Side-nav when on-page space is a priority. The Flexible Side-nav has both an expanded and collapsed state. The expanded state contains both icons and corresponding L2 text, while the collapsed state shows only the icon. The side-nav expands on hover. It can also be expanded (to a fixed state) or collapsed by clicking on the chevron icon at the bottom of the panel.
 
 In a Flexible Side-nav, each L2 link and L2 category list has its own corresponding icon. Side-nav icons are stacked equidistant from each other in the collapsed state, and when expanded are aligned with their corresponding L2 labels.
 
+A flexible side-nav is more friendly to responsive designs than a fixed nav. Fixed nav is best for products that have a minimum width of 672px. Consider using a flexible nav for a fully responsive solution.
+
+Use a flexible side-nav when:
+
+- more space is needed for page content
+- a fully responsive solution is needed 
+
 #### Fixed Side-nav
 
 A fixed Side-nav contains only L2 Links and L2 Category lists, no icons, and cannot be collapsed.
 
+Use a fixed side-nav when:
+
+- icons cannot clearly describe the categories
+- designs are for desktop only
+
 ![Header](images/side-nav-panel.png)
```

Comment:
Incorrect alt text. The text between the [ ] should describe the image. It is the text this will be read by the screen reader.

Reply:
This image also belongs under "Flexible side-nav" not "Fixed" because it's an example of the Flexible component. We can create one for fixed too if we need it.